consensus/istanbul/backend/backend.go

```diff
@@ -712,3 +712,13 @@ func (sb *Backend) ValidatorAddress() common.Address {
 	}
 	return localAddress
 }
+
+func (sb *Backend) ConnectToVals() {
+	// If this is a proxy, then refresh the val peers.  Note that this will be done within Backend.Start
```

Comment:
nit: if you remove the conditional and renamed it `InitiateValidatorConnections` or something, would it be appropriate to call this in `eth/handler.go` and remove the call in the consensus engine start?

Reply:
The reason for the two separate invocations of `RefreshValPeers` is that in `eth/handler.go`, I don't know what type of node it will be.   I could modify the `eth` object so that it contains a field to store that info, but my plan is to supersede the changes in this PR with changes for this issue:  https://github.com/celo-org/celo-blockchain/issues/607.

My plan is to have a separate validator peer manager that will include a goroutine that will be continuously monitoring the validator connections and connect/disconnect as needed.

Reply:
Makes total sense, thanks for the explanation.